doc/releasenotes/12_0_0_release_notes.md

```diff
@@ -31,8 +31,6 @@ zone1-0000000103 sourcekeyspace 0 rdonly 192.168.0.134:15103 192.168.0.134:17103
 
 ## Deprecations
 
-The command `vtctl VExec` is deprecated and removed. All Online DDL commands should run through `vtctl OnlineDDL`.
-
```

Comment:
This should be reverted. See https://github.com/vitessio/vitess/pull/9228#discussion_r750829720

Reply:
Can you start a 12_0_1_release_notes.md and add a line about reverting the deprecation?

Reply:
Done.